x-pack/plugins/triggers_actions_ui/public/application/sections/actions_connectors_list/components/actions_connectors_home.test.tsx

```diff
@@ -143,6 +143,39 @@ describe('ActionsConnectorsHome', () => {
     expect(documentationButton).toBeEnabled();
   });
 
+  it('show "Create connector" and "Documentation" buttons when on Connectors Edit tab', async () => {
+    const props: RouteComponentProps<MatchParams> = {
+      history: createMemoryHistory({
+        initialEntries: ['/connectors/1'],
+      }),
+      location: createLocation('/connectors/1'),
+      match: {
+        isExact: true,
+        path: '/connectors/1',
+        url: '',
+        params: {
+          section: 'connectors',
+        },
+      },
+    };
+
+    render(
+      <IntlProvider locale="en">
+        <Router history={props.history}>
+          <QueryClientProvider client={queryClient}>
+            <ActionsConnectorsHome {...props} />
+          </QueryClientProvider>
+        </Router>
+      </IntlProvider>
+    );
+
+    const createConnectorButton = await screen.findByRole('button', { name: 'Create connector' });
```

Comment:
suggestion: use findByTestId instead vs label text.

Reply:
I used above test just with different params of route. It is better to either keep consistency or change it everywhere. As we are not codeowners I would prefer to keep it consistent